docs/tools/piral-inspector.md

```diff
@@ -4,4 +4,203 @@ title: Piral Inspector
+:::
+
+## Using Piral Inspector
+
+Piral Inspector offers a wide variety of capabilities regarding Piral Instance and its corresponding Pilets.
+
+The add-on/extension operates on a locally in debug mode running Piral Instance, and it directly interacts with it.
+
+The following sections are shown inside the extension:
+- Information about Piral Instance
+- Available Pilets
+- Add Pilets
+- Registered Routes
+- Events
+- State Container
+
+All of these sections will documented in the following chapters:
+
+### Piral Instance
+In the very left top, the extension shows the name and the version of the currently running Piral Instance.
+Besides that, in the very right top, a click on the icon allows the user to adapt the extension's settings to its needs.
+
+![PiralInstanceToolPiralInspector](../diagrams/piralinspector_tool_piralinstance.png)
+
+In the debug setting, the user can activate/deactivate the following options:
+
+![DebugSettingsToolPiralInspector](../diagrams/piralinspector_tool_debugsettings.png)
+
+#### State Container Logging
+Activates/Deactivates the logging of changes in the state container to the Developer Tools console.
+
+#### Load available Pilets
+Activates/Deactivates the loading of the available Pilets.
+::: warning: Only for Pilets
+This feature has an impact only when the emulator is used, i.e., when `pilet debug` is running.
+:::
+
+#### Full refresh on change
+Refreshes the current browser page in case there was a change in the debugged Pilet.
+::: warning: Only for Pilets
+This feature has an impact only when the emulator is used, i.e., when `pilet debug` is running.
+:::
+
+#### Visualize component origins
+Activates the option to visualize the origin of the modules as another icon in Piral Instance section:
+
+![DebugSettingsVisCompOriginToolPiralInspector](../diagrams/piralinspector_tool_visualizecomponentoriginicon.png)
+
+After clicking on the icon, the origin of each single tile will be displayed:
+
+![DebugSettingsVisCompOriginToolPiralInspector](../diagrams/piralinspector_tool_visualizecomponentorigintile.png)
+
+The origin will also be displayed when hovering over a tile.
+
+
+### Available Pilets
+The Piral Inspector lists all the currently available Pilets. Like illustrated in the table below.
+
+![AvailablePiletsToolPiralInspector](../diagrams/piralinspector_tool_availablePilets.png)
+
+By clicking on the switch button on the left, the user can activate and deactivate each Pilet.
+
+With a click on the "X" on the right, the user can remove each Pilet individually.
+
+### Add Pilets
+There are two options to load additional Pilets into the currently running Piral Instance:
+
+#### Add a feed address
+Enter a feed address into the text field and click on the "Add" button.
+In case you chose a feed you have access to, the corresponding Pilets will be loaded and shown in the tile section and the list of available Pilets. Per default, the new Pilets will be active.
```

Comment:
Also here there are many newlines missing. Just orient on how the other md files have been written.

Reply:
Okay.